Microsoft.Toolkit.Uwp.UI.Controls/TokenizingTextBox/TokenizingTextBox.xaml

```diff
@@ -12,6 +12,7 @@
     <Thickness x:Key="TokenizingTextBoxPresenterMargin">0,0,6,0</Thickness>
     <x:Double x:Key="TokenizingTextBoxTokenSpacing">2</x:Double>
     <Thickness x:Key="TextControlBorderThemeThicknessFocused">2</Thickness>
+    <x:Double x:Key="TokenizingTextBoxIconFontSize">16</x:Double>
```

Comment:
This is actually ignored as we don't have a good way to access it from the code-behind of the templated control (another helper in the future?).  I'm not sure if that's what the [ComponentResourceKey](https://docs.microsoft.com/en-us/dotnet/framework/wpf/advanced/componentresourcekey-markup-extension?redirectedfrom=MSDN) was for in WPF (I never used that in my days there).

I can add a comment linking to the hard-coded value on line 110 below in TTB.ASB.cs.